crates/cheatnet/src/state.rs

```diff
@@ -210,8 +237,11 @@ impl CheatcodeState {
     }
 }
 
-impl Default for CheatcodeState {
-    fn default() -> Self {
-        Self::new()
-    }
-}
+// impl Default for CheatnetState {
```

Comment:
Should this be deleted?

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/753/commits/f02c662ca4cacb71b1f25f5a752357bf684465cc